packages/manager/apps/dedicated/client/app/account/user/security/u2f/delete/user-security-u2f-delete.controller.js

```diff
@@ -77,34 +73,7 @@ export default [
      * Init.
      * @return {Promise}
      */
-    $scope.init = () => {
-      let promise = $q.when(true);
-      if ($scope.u2f.u2fAccount.status === 'enabled') {
-        $scope.u2f.isUpdating = true;
-        promise = DoubleAuthU2fService.challenge(
-          $scope.u2f.u2fAccount.id,
-          $scope.u2f.u2fAccount.status,
-        );
-      }
-      $scope.u2f.isLoading = true;
-      $scope.u2f.hasError = false;
-      return promise
-        .then(() => $scope.deleteDoubleAuthU2fKey())
-        .catch((err) => {
-          let key = null;
-          if (err.request.errorCode === 3 || err.request.errorCode === 4) {
-            key = `user_account_security_double_auth_type_u2f_add_error_code_${err.request.errorCode}`;
-          } else {
-            key = 'user_account_security_double_auth_type_u2f_add_error';
-          }
-          $scope.u2f.isLoading = false;
-          $scope.u2f.hasError = true;
-          Alerter.error($translate.instant(key), 'doubleAuthAlertU2fDelete');
-        })
-        .finally(() => {
-          $scope.u2f.isLoading = false;
-        });
-    };
+    $scope.init = () => {};
```

Comment:
**nitpick:** Empty function can be removed. 
Also don't forget to remove it from Line number 10 on `packages/manager/apps/dedicated/client/app/account/user/security/u2f/delete/user-security-u2f-delete.html`
```suggestion
   
```

Reply:
Done